packages/react/kibana_context/theme/README.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: react/context/theme
+slug: /react/context/theme
+title: React Context - Theme
+description: This context allows a one to alter the theme for a given component.  This is likely to be the context that is used most often.
+tags: ['shared-ux', 'react', 'context']
+date: 2023-07-25
+---
+
+## Description
+
+This package contains a "theming" context for Kibana.  A corollary to EUI's `EuiThemeProvider`, it uses Kibana services to ensure the EUI Theme is customized correctly.
+
+Up until now, there has been some confusion as to the difference between `EuiThemeProvider` and `EuiProvider`.  They've been used interchangeably, which created some unnoticed
+side effects.  In addition, _nesting_ of `EuiThemeProvider` has led to additional nodes being rendered to the DOM.
+
+This context allows us to have a single source of truth for the theme in Kibana, and ensures that the theme is applied correctly.  It also abstracts away any updates or changes
+made to the `EuiThemeProvider` in the future.
+
```

Comment:
nit: could provide code snippet here as in the other readme files to make it clear what the preferred usage pattern is

Reply:
Oh, I thought I had.  Will fix.